.github/workflows/on-code-change.yml

```diff
@@ -0,0 +1,31 @@
+name: On Code Change
```

Comment:
As the job is triggered on `push` maybe we can rename this workflow to fit what it does ?

Reply:
It's triggered on push only because I didn't finish the migration and I test it each time I change something